src/codeflare_sdk/ray/cluster/cluster.py

```diff
@@ -744,6 +802,20 @@ def _create_resources(yamls, namespace: str, api_instance: client.CustomObjectsA
     )
 
 
+def _apply_resources(
```

Comment:
Could you just use a [partial](https://www.geeksforgeeks.org/partial-functions-python/) instead of pulling this into it's own function?